usaspending_api/download/v2/base_download_viewset.py

```diff
@@ -125,6 +122,47 @@ def _get_status_url(self, file_name: str) -> str:
 
         return f"{protocol}://{host}/api/v2/download/status?file_name={file_name}"
 
+    @staticmethod
+    def _get_cached_download(
+        ordered_json_request: str, download_types: Optional[List[str]] = None
+    ) -> Optional[QuerySet]:
+        external_data_type_name_list = []
+
+        # External data types that directly affect download results
+        if download_types:
+            if "elasticsearch_awards" in download_types:
+                external_data_type_name_list.append("es_awards")
+            elif "elasticsearch_transactions" in download_types:
+                external_data_type_name_list.append("es_transactions")
+        else:
+            external_data_type_name_list.extend(["fpds", "fabs", "es_transactions", "es_awards"])
+
+        external_data_type_id_list = [
+            id for name, id in EXTERNAL_DATA_TYPE_DICT.items() if name in external_data_type_name_list
+        ]
+
+        # Clear the download "cache" based on most recent date in the list of relevant ExternalDataLoadDate objects
+        updated_date_timestamp = ExternalDataLoadDate.objects.filter(
+            external_data_type_id__in=external_data_type_id_list
+        ).aggregate(Max("last_load_date"))["last_load_date__max"]
+
+        # Conditional put in place for local development where the external dates may not be defined
+        if updated_date_timestamp:
```

Comment:
Minor request: Can you take the return out of this condition by setting a default value then updating it in the condition?